### DISCUSSION TITLE
After deploying a symfony project to cloudpanel how do you make it work?

### DISCUSSION THREAD

**Discussion by IamLiam09** [symfony/symfony#52492] [Q&A]
After deploying a symfony project to cloudpanel how do you make it work?

**Comment by ghost** [comment]
Symfony doesn't care where you deploy the app. Symfony app is just like every other PHP app i.e. you have a front controller like `public/index.php` and you configure your PHP interpreter is use to execute it.

**Comment by 94noni** [comment]
Indeed
And if « it does not work » you may need to read the documentation on how to deploy symfony to production and if already done, check your production logs as well :)
